sdk/identity/azure-identity/README.md

```diff
@@ -178,6 +174,7 @@ The credential types in Azure Identity differ in the types of AAD identities the
 </table>
 
 
+
```

Comment:
Remove all extra lines.  There should be at most one empty line in between sections.

Reply:
That applies to above the table as well.

Reply:
Thank you very much for patiently reviewing and suggesting changes. Will keep these things in mind henceforth.